### ISSUE TITLE
False positive: "all caps string key" error

### ISSUE THREAD

**Issue by Bastianleaf** [rebelappstudio/rebellion#13]
## Problem
When processing a `.arb` file with **Rebellion**, the following error occurs:

```bash
./assets/l10n/app_es.arb: all caps string key "plannerDestination"
```

## Steps to Reproduce
Use a .arb file with the following content:

```json
{
  "plannerDestination": "A {destination}",
  "@plannerDestination": {
    "placeholders": {
      "destination": {
        "type": "String"
      }
    }
  }
}
```
- Run Rebellion to process the file.
- Observe the error.

The key plannerDestination should not trigger an "all caps string key" error since it is correctly formatted.

**Comment by aednlaxer** [comment]
Hi,

Thanks for the bug report! I guess strings with placeholders should never be marked as errored since we can't guarantee what placeholder is going to be used for. Also, #5 should provide an option to ignore certain rules like `all_caps` in cases like that but I believe in this case a proper fix would be preferred over ignoring rules

**Comment by aednlaxer** [comment]
Fixed in #15 and released in [v0.1.0](https://pub.dev/packages/rebellion/versions/0.1.0). Single character uppercase strings should not be marked as errors
